pack/df.json

```diff
@@ -511,7 +511,7 @@
         "deck_limit": 3,
         "faction_code": "neutral-runner",
         "faction_cost": 0,
-        "flavor": "I didn't say your hopper should be hauling garbage. I said it should be hauled away as garbage.",
+        "flavor": "I didn't say your hopper should be hauling garbage. I said it should be hauled away <strong>as</strong> garbage.",
```

Comment:
Won't using `<strong>` here bold the text, rather than un-italicise it like on the card image?

For example, check QPM:

https://github.com/Alsciende/netrunner-cards-json/blob/master/pack/dad.json#L114